playground/frontend/integration_test/standalone_editing_test.dart

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:playground_components_dev/playground_components_dev.dart';
+
+import 'common/common.dart';
+import 'common/common_finders.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Testing editing code', (WidgetTester wt) async {
+    await init(wt);
+    await _checkAutocomplete(wt);
+    await _editingAndResettingChanges(wt);
+    await _checkCodeHighlighting(wt);
+    await _codeBlockFoldingTest(wt);
+  });
+}
+
+Future<void> _checkAutocomplete(WidgetTester wt) async {
+  final codeController = wt.findOneCodeController();
+  final sSuggestions = await codeController.autocompleter.getSuggestions('sdk');
+  print(sSuggestions.map((e) => "'$e'").join(', '));
+  expect(
+    sSuggestions,
+    [
+      'sdkHttpMetadata',
+      'sdkHttpMetadataWithoutHeaders',
+      'sdkHttpResponse',
+      'sdkHttpResponseWithoutHeaders'
+    ],
+  );
```

Comment:
1. Set the entire text to "sdk".
2. Send a shortcut to open the suggestions box (create it).
3. Check the content of the suggestion box.

Reply:
done